src/Orleans.Testing.Utils/LocalCluster.cs

```diff
@@ -35,13 +35,9 @@ public Task StopAsync(CancellationToken cancellationToken = default)
             return _siloHost.StopAsync(cancellationToken);
         }
 
-        public async void Dispose()
+        public void Dispose()
         {
-            using (var cts = new CancellationTokenSource())
-            {
-                cts.Cancel();
-                await StopAsync(cts.Token);
```

Comment:
This trick does not work anymore. We have to wait for it to complete now.

Reply:
Eww